activerecord/lib/active_record/railties/databases.rake

```diff
@@ -100,11 +100,13 @@ db_namespace = namespace :db do
 
       file_list =
           ActiveRecord::Tasks::DatabaseTasks.migrations_paths.flat_map do |path|
-            # match "20091231235959_some_name.rb" and "001_some_name.rb" pattern
-            Dir.foreach(path).grep(/^(\d{3,})_(.+)\.rb$/) do
-              version = ActiveRecord::SchemaMigration.normalize_migration_number($1)
+            Dir.foreach(path).select do |file|
+              ActiveRecord::Migrator.match_to_migration_filename?(File.basename(file))
+            end.map do |file|
```

Comment:
Rather than using `select` here why not just use this:

``` ruby
Dir.foreach(path) do |file|
  next unless ActiveRecord::Migrator.match_to_migration_filename?(File.basename(file))

  # do stuff
end
```